shared/actions/settings.js

```diff
@@ -66,6 +93,135 @@ function * saveNotificationsSaga (): SagaGenerator<any, any> {
   }
 }
 
+function * reclaimInviteSaga (invitesReclaimAction: InvitesReclaim): SagaGenerator<any, any> {
+  const {inviteId} = invitesReclaimAction.payload
+  try {
+    yield call(apiserverPostRpcPromise, {
+      param: {
+        endpoint: 'cancel_invitation',
+        args: [{key: 'invitation_id', value: inviteId}],
+      },
+    })
+    yield put(({
+      type: Constants.invitesReclaimed,
+      payload: undefined,
+      error: false,
```

Comment:
> shouldn't need `error: false`

It's a bit of a common pattern:
```js
 λ git grep "error: false" | wc -l
14
```
I think I like it because `payload: undefined` looks unfriendly; it makes it not obvious through scanning that the code block is a success case.  If we could skip `payload: undefined` *too*, I'd feel better about skipping error.

What do you think?  If you don't like `error: false`, perhaps we should have a PR for removing it everywhere?